firebase/functions/index.js

```diff
@@ -0,0 +1,50 @@
+// @license
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+const functions = require('firebase-functions');
+const request = require('request-promise-native');
+require('babel-polyfill');
+const cors = require('cors')({origin: '*'});
```

Comment:
I wondered whether in prod we should constrain this to the specific route we know we need for each service, to reduce evil doer impact on our bits, unlikely but if easy to prevent may as well. I saw route specific config info here https://www.npmjs.com/package/cors

Reply:
Route as in endpoint, or something else?

If it's route as in endpoint, in this file it is restricted only to specific endpoints, but we happen to need it for all of them. It's only enabled in methods that include the call to `cors()`.

Reply:
I do have some thoughts to put in to restrict access at least a little bit before we put auth in, I'll add that before putting up the final PR.